python/src/scipp/plot/plot_2d.py

```diff
@@ -17,120 +20,116 @@
 import warnings
 
 
-def plot_2d(scipp_obj_dict=None,
-            axes=None,
-            values=None,
-            variances=None,
-            masks=None,
-            filename=None,
-            figsize=None,
-            mpl_axes=None,
-            aspect=None,
-            cmap=None,
-            log=False,
-            vmin=None,
-            vmax=None,
-            color=None,
-            logx=False,
-            logy=False,
-            logxy=False):
+def plot_2d(to_plot: List[PlotRequest]):
     """
     Plot a 2D slice through a N dimensional dataset. For every dimension above
     2, a slider is created to adjust the position of the slice in that
     particular dimension.
     """
+    # This check is duplicated to assist the IDE to deduct we are using a
```

Comment:
I wouldn't include anything which is not necessary for the function of the plotting (e.g/ to help IDEs).

Reply:
follow-up: you mentioned the comment was out-dated